noxfile.py

```diff
@@ -0,0 +1,105 @@
+import nox
+import os
+import sys
+
+
+@nox.session(name="pybamm-requires")
+def run_pybamm_requires(session):
+    if sys.platform != "win32" or sys.platform != "darwin":
+        session.install("wget", "cmake")
+        session.run("python", "scripts/install_KLU_Sundials.py")
+        session.run(
+            "git",
+            "clone",
+            "https://github.com/pybind/pybind11.git",
+            "pybind11/",
+            external=True,
+        )
+
+
+@nox.session(name="coverage")
+def run_coverage(session):
+    homedir = os.getenv("HOME")
+    session.env["SUNDIALS_INST"] = session.env.get("SUNDIALS_INST", f"{homedir}/.local")
+    session.env[
+        "LD_LIBRARY_PATH"
+    ] = f"{homedir}/.local/lib:{session.env.get('LD_LIBRARY_PATH')}"
```

Comment:
Hmm, why aren't we setting these variables in `run_pybamm_requires`?

Reply:
Since these were not also declared in `tox : [pybamm-requires]` I'm setting up these variables wherever required.

Reply:
I think it would be better to just add them in `run_pybamm_requires`.

Reply:
I've added them in `run_pybamm_requires` as well. 
You don't want me to set them globally, do you 👀.